src/data_vis.ts

```diff
@@ -96,6 +96,8 @@ class DataVisModel {
   private impactData = ko.observable(getDataSet(initMethod, montaguDataSets).data);
   private yearMethod = ko.observable(initMethod);
 
+  private plotColours = ko.observable(this.isPrivate() ? {...plotColours, ...legacyColours} : plotColours);
```

Comment:
I think 'isPrivate' is going to be replaced by two flags to support the paper 2 app - fullData (true of legacy funders app and paper 2 app) and stochasticData (true of public paper 1 app and paper 2 app). We can update this property when that happens.